main/res/values/styles.xml

```diff
@@ -73,11 +74,17 @@
         <item name="android:textColor">@color/colorTextActionBar</item>
     </style>
 
+
     <style name="settings_actionbar_theme" parent="@style/ThemeOverlay.MaterialComponents.ActionBar">
         <!-- Customize color of back arrow -->
         <item name="android:textColorPrimary">@color/colorTextActionBar</item>
     </style>
 
+    <style name="menu_item">
+        <item name="android:tint">@color/colorIconMenu</item>
+        <item name="iconTint">@color/colorIconMenu</item>
+    </style>
+
```

Comment:
I tried that and it didn't work, but probably I used the wrong attribute...

Reply:
Somewhere I've read that `textColorPrimary` it the attribute which needs to be overwritten. But I might be wrong. 

As nearly everything in c:geo is WIP currently, I guess we could merge this PR without  fine tuning if you prefer. Just give a hint if you think you're finished. 

For everything which is not yet solved, we just need to create follow up issues...

Reply:
> Ok, I tried it on the action itself, not.on the while actionbar 

Yes, probably it needs to be set in the `actionBarStyle` which is the one used in the global cgeo theme

Reply:
Seems, that the problem is using a definition @color/colorIcon inside the svg. This will not correctyl be resolved if it is set to ?android:textColorPrimary. If I set ?android:textColorPrimary directly in the svg it works.
But using it for fillColor the preview does not work, while using it for tint, the preview works - strange.
So I will change the tint inside the svg to ?android:textColorPrimary, so I can remove the tint from the styles.xml